src/feature/Users/components/EmployeeTable/index.jsx

```diff
@@ -19,7 +20,11 @@ const EmployeeTable = (props) => {
       </thead>
       <tbody>
         {employees.map((employee) => (
-          <EmployeeRow key={employee.id} {...employee} />
+          <EmployeeRow
+            key={employee.id}
+            {...employee}
```

Comment:
Don't spread the data like so, let's have a single prop `employee`, then pass the data object.

```suggestion
            employee={employee}
```